app/models/action.rb

```diff
@@ -0,0 +1,5 @@
+class Action < Sequel::Model
+  many_to_one :effect
+  many_to_many :attributes
```

Comment:
many_to_one and many_to_many are 'class methods' from Sequel::Model. You can call them in the body of a subclass. They set up associations. many_to_one :effect should create an instance method for Action called effect, which will return the effect corresponding to the effect_id column in the actions table. It also adds methods for setting the effect, or for creating an effect linked to this action. many_to_many does the same, but for a collection of Attribute objects, instead of a single Effect object.